users/views.py

```diff
@@ -49,3 +51,26 @@ def post(self, request):
 
         except ValidationError as error:
             return JsonResponse({"message": error.message}, status=error.code)
+
+
+class SignInView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            email    = data['email']
+            password = data['password']
+
+            user     = User.objects.get(email = email)
+
+            access_token = jwt.encode({'id':user.id, 'exp': datetime.utcnow()+timedelta(days=1)},SECRET_KEY,algorithm=ALGORITHM)
+
+            if bcrypt.checkpw(password.encode('utf-8'),user.password.encode('utf-8')):
+                return JsonResponse({'messasge':'SUCCESS','ACCESS_TOKEN':access_token}, status=200)
+            return JsonResponse({"message":"INCORRECT_PASSWORD"},status=401)
+
+        except KeyError:
+            return JsonResponse({"message":"KEY_ERROR"},status=400)
+            
+        except User.DoesNotExist:
+            return JsonResponse({"message":"NOT_REGISTERED_EMAIL"},status=401)
```

Comment:
```suggestion
            return JsonResponse({"message":"INVALID_USER"},status=401)
```
로그인 실패시에 비밀번호가 틀렸는지 아이디가 틀렸는지 모르게 하는 것이 보안상으로 더 좋습니다.